types/vanillajs-datepicker/DateRangePicker.d.ts

```diff
@@ -0,0 +1,24 @@
+import Datepicker from './Datepicker';
+
+export interface DateRangePickerOptions {
+    inputs?: HTMLElement[];
+    allowOneSidedRange?: boolean;
+}
+export default class DateRangePicker {
+    constructor(element: HTMLElement, options?: DateRangePickerOptions);
+    allowOneSidedRange: boolean;
+    element: HTMLElement;
+    inputs: HTMLElement[];
+    datepickers: Datepicker[];
+
+	get dates(): Date[] | undefined;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/vanillajs-datepicker/DateRangePicker.d.ts;
  Line: 14;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.